typescript/src/entities/user.ts

```diff
@@ -49,6 +49,9 @@ export class User extends Entity {
   @User.property()
   public isSuperUser?: boolean;
 
+  @User.property()
+  public userType?: number;
```

Comment:
userType?: UserType

I will approve it, depends on whether u want to change it

Reply:
I did think of this but I checked the convention set for the productType attr on product and it doesn't take advantage of the ProductType constant.